src/main/scala/com.snowplowanalytics.iglu/client/SchemaKey.scala

```diff
@@ -34,7 +34,7 @@ import ProcessingMessageMethods._
  */
 object SchemaKey {
 
-  private val SchemaUriRegex = "^iglu:([a-zA-Z0-9-_.]+)/([a-zA-Z0-9-_]+)/([a-zA-Z0-9-_]+)/((?:[0-9]+-)?[0-9]+-[0-9]+)$".r
+  private val SchemaUriRegex = "^iglu:([a-zA-Z0-9-_.]+)/([a-zA-Z0-9-_]+)/([a-zA-Z0-9-_]+)/([0-9]+-[0-9]+-[0-9]+)$".r
```

Comment:
We should probably add a ticket for this change, since it is technically a breaking change.